NodeJS/__integration__/AgentClient.it.ts

```diff
@@ -1,62 +1,81 @@
-       expect(result).toBeTruthy()
-    })
-    test("waitForAgentDeletion should return true", async () => {
-        let result = await agentClient.waitForAgentDeletion("agentNotExists")
-        expect(result).toBeTruthy()
-     })
-     test("getSkillIds should return skillIds", async () => {
-         let skillIds = await agentClient.getSkillIds()
-         expect(skillIds.length).toBeGreaterThan(0)
-     })
-     test("getSkillNumbers should return skillNumbers", async () => {
-         let skillNumbers = await agentClient.getSkillNumbers()
-         expect(skillNumbers.length).toBeGreaterThan(0)
-     })
-     xtest("existsAgentByLoginId should return true when agent exists", async () => {
-        let exists = await agentClient.existsAgentByLoginId(7300000100)
-        expect(exists).toBeTruthy()
-     })
-     xtest("waitForAgentCreation should return true when agent exists", async () => {
-        let exists = await agentClient.waitForAgentCreation(7300000100)
-        expect(exists).toBeTruthy()
-     })
-     xtest("waitForAgentCreation should return false when agent does not exists", async () => {
-        let exists = await agentClient.waitForAgentCreation(3337300000100)
-        expect(exists).toBeFalsy()
-     })
-     xtest("existsAgentByUsername returns when agent exists by username", async () => {
-        let exists = await agentClient.existsAgentByUsername("ddksgy3dnr")
-        expect(exists).toBeTruthy()
-     })
-     xtest("existsStationForAgent should return false when agent has no station", async () => {
-        let exists = await agentClient.existsStationForAgent("ddksgy3dnr")
-        expect(exists).toBeFalsy()
-     })
-     xtest("getAgent should return agent and station", async () => {
-         let agentAndStation = await agentClient.getAgent("ddksgy3dnr")
-         expect(agentAndStation.agent).toBeDefined()
-         expect(agentAndStation.station).toEqual({})
-     })
-     xtest("createStationIfNotExists should return true", async ()=> {
-         let result = await agentClient.createStationIfNotExists("ddksgy3dnr", "2")
-         expect(result).toBeTruthy()
-     })
-     test("getUserToken should return token", async () => {
-        let result = await agentClient.getUserToken("yangadmin1")
-        expect(result).toEqual(token)
-     })
+   beforeEach(async () => {
+      agentClient = await createAgentClient("http://localhost:8081", token)
+   });
+   test("waitForStationDeletion should return true", async () => {
+      let result = await agentClient.waitForStationDeletion("agentNotExists")
+      expect(result).toBeTruthy()
+   })
+   test("waitForAgentDeletion should return true", async () => {
+      let result = await agentClient.waitForAgentDeletion("agentNotExists")
+      expect(result).toBeTruthy()
+   })
+   test("getSkillIds should return skillIds", async () => {
+      let skillIds = await agentClient.getSkillIds()
+      console.log(skillIds)
```

Comment:
remove `console.log`

Reply:
It is ok to keep it in an integration test.